.github/CODEOWNERS

```diff
@@ -56,38 +32,49 @@ install_build_dependencies.sh @openvinotoolkit/openvino-scripts-maintainers @ope
 /src/common/preprocessing/ @openvinotoolkit/openvino-ie-maintainers
 /src/common/util/ @openvinotoolkit/openvino-ie-maintainers
 /thirdparty/ @openvinotoolkit/openvino-ie-maintainers
-.gitmodules @openvinotoolkit/openvino-ie-maintainers
+/.gitmodules @openvinotoolkit/openvino-ie-maintainers
 
 /src/bindings/python/ @openvinotoolkit/openvino-ie-python-api-maintainers
 /src/bindings/c/ @openvinotoolkit/openvino-c-api-maintainers
 /src/common/*transformations/  @openvinotoolkit/openvino-ie-transformations-maintainers
 /src/core/  @openvinotoolkit/openvino-ngraph-maintainers
 
+# OpenVINO Samples:
+/samples/c/ @openvinotoolkit/openvino-samples-maintainers @openvinotoolkit/openvino-c-api-maintainers
+/samples/cpp/  @openvinotoolkit/openvino-samples-maintainers @openvinotoolkit/openvino-maintainers
+/samples/python/ @openvinotoolkit/openvino-samples-maintainers @openvinotoolkit/openvino-ie-python-api-maintainers
+/thirdparty/zlib/ @openvinotoolkit/openvino-samples-maintainers
+/thirdparty/json/ @openvinotoolkit/openvino-samples-maintainers
+/thirdparty/gflags/ @openvinotoolkit/openvino-samples-maintainers
+/thirdparty/cnpy/ @openvinotoolkit/openvino-samples-maintainers
+
+# OpenVINO Func Tests:
+/src/tests/  @openvinotoolkit/openvino-ie-tests-maintainers @openvinotoolkit/openvino-ie-test-developers
+/src/tests_deprecated/  @openvinotoolkit/openvino-ie-tests-maintainers @openvinotoolkit/openvino-ie-test-developers
+/src/frontends/tests/frontend/shared/ @openvinotoolkit/openvino-ie-tests-maintainers
+/thirdparty/gtest/ @openvinotoolkit/openvino-ie-tests-maintainers
+
 # OpenVINO CPU:
 /src/plugins/intel_cpu/  @openvinotoolkit/openvino-ie-cpu-maintainers @openvinotoolkit/openvino-ie-cpu-developers
 /src/common/snippets/ @openvinotoolkit/openvino-ie-cpu-maintainers
-/src/tests/**cpu** @openvinotoolkit/openvino-ie-cpu-maintainers
+/src/tests/**/cpu/ @openvinotoolkit/openvino-ie-cpu-maintainers
```

Comment:
I guess that already all cpu tests should be located inside the plugin folder

Reply:
Looks like no - https://github.com/openvinotoolkit/openvino/tree/master/src/tests/functional/plugin/cpu
It's a mistake and those files need to be moved.

So, I've removed this filter for CPU plugin.

Reply:
https://github.com/openvinotoolkit/openvino/pull/14887